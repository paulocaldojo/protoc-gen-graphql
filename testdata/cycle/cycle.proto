syntax = "proto3";

package protoc_gen_graphql.test.cycle;

option go_package = "github.com/paulocaldojo/protoc-gen-graphql/testdata/cycle";

message A {
  B b = 1;
}

message B {
  C c = 1;
}

message C {
  A a = 1;
}
