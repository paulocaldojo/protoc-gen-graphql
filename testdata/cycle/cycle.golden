# DO NOT EDIT! Generated by protoc-gen-graphql.

type ProtocGenGraphqlTestCycle_A {
  b: ProtocGenGraphqlTestCycle_B!
}

type ProtocGenGraphqlTestCycle_B {
  c: ProtocGenGraphqlTestCycle_C!
}

type ProtocGenGraphqlTestCycle_C {
  a: ProtocGenGraphqlTestCycle_A!
}
